README.md

```diff
@@ -19,7 +19,7 @@
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](https://github.com/datalad/datalad/blob/master/CODE_OF_CONDUCT.md)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.808846.svg)](https://doi.org/10.5281/zenodo.808846)
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-46-orange.svg?style=flat-square)](#acknowledgements)
+[![All Contributors](https://img.shields.io/badge/all_contributors-46-orange.svg?style=flat-square)](#contributors-)
```

Comment:
eh, I have fixed it to correspond to the one we have but didn't realize that it came after the ALL-CONTRIBUTERS-BADGE:START and that it is hardcoded: https://github.com/all-contributors/cli/blob/master/src/init/__tests__/add-badge.js#L8 .  So we would just need to either add section `Contributors` below or add some dummy `<span id="contributors"/>` which might accomplish the same drill.  Why one would you prefer @datalad/developers ?

Reply:
I don't care much, tbh. From my point of view a dedicated contributors section is fine.

Reply:
ok, added subsubsection below, tested in preview -- works as intended. Let's proceed